src/ui/common/src/components/notifications/NotificationListItem.tsx

```diff
@@ -126,18 +112,25 @@ export const NotificationListItem: React.FC<Props> = ({
         <Box>
           <Typography
             variant="body1"
-            sx={{ fontWeight: 'medium', color: 'gray600' }}
+            sx={{
+              fontWeight: 'light',
+              color: 'gray600',
```

Comment:
Can we change the weight back to medium here?

Should be gray.600

Also, the comment below in line 121 no longer reflects what is happening. Can you fix that up too?